schema/deploy/type_facility_application_status.sql

```diff
@@ -0,0 +1,25 @@
+-- Deploy ggircs-portal:type_facility_application_status to pg
+-- requires: schema_ggircs_portal
+-- requires: table_facility
+-- requires: table_application
+-- requires: type_ciip_application_status
+
+begin;
+
+create type ggircs_portal.facility_application_status as (
```

Comment:
This type needs to be renamed as it is confusing (sounds like it's the status of the relation between a facility and an application). `facility_search_result` would be better.